pep-0692.rst

```diff
@@ -463,13 +534,15 @@ previous example::
     >>> def foo(**kwargs: **Movie): ...
     ...
     >>> foo.__annotations__
-    {'kwargs': **Movie}
+    {'kwargs': Unpack[Movie]}
 
-The double asterisk syntax should call the ``__unpack__`` special method on
-the object it was used on. This means that ``def foo(**kwargs: **T): ...`` is
-equivalent to ``def foo(**kwargs: T.__unpack__()): ...``. In addition,
-``**Movie`` in the example above is the ``repr`` of the object that
-``__unpack__()`` returns.
+To accomplish this, we propose a new dunder called ``__typing_kwargs_unpack__``.
+The double asterisk syntax should result in a call to the
+``__typing_kwargs_unpack__`` special method on an object it was used on.
+This means that ``def foo(**kwargs: **T): ...`` is equivalent to
+``def foo(**kwargs: T.__typing_kwargs_unpack__()): ...``.
+``TypedDict`` is the only type in the standard library that will implement
+``__typing_kwargs_unpack`` and it should return ``Unpack[self]``.
```

Comment:
I'd make this less strong -- I don't want the PEP to read as forbidding to implement that dunder on other types if the need arises. It should be phrased as an expectation.

Reply:
```suggestion
``TypedDict`` is the only type in the standard library that will implement
``__typing_kwargs_unpack__`` and it should return ``Unpack[self]``.
```

Its also not a dunder at all here, due to a typo